src/EPR.Payment.Facade.Common/RESTServices/Payments/HttpPaymentsService.cs

```diff
@@ -23,8 +20,6 @@ public HttpPaymentsService(
                 config.Value.EndPointName ?? throw new ArgumentNullException(nameof(config), ExceptionMessages.PaymentServiceEndPointNameMissing))
         {
             _httpContextAccessor = httpContextAccessor ?? throw new ArgumentNullException(nameof(httpContextAccessor));
```

Comment:
I understand this is a clean up activity

Reply:
Yes, I noticed it was not needed in HttpRegistrationFeesService, so cleaned up HttpPaymentsService